### ISSUE TITLE
b2b checkout test

### ISSUE THREAD

**Issue by WeizhengSap** [SAP/spartacus#7907]
To QA:
1. first make sure the b2c checkout work well (same as before)
2. switch to b2b, make sure each step work well
3. change payment type, the checkout data will be cleared, and customer redirect to first step
4. change language works well in each step
5. customer can checkout with ACCOUNT
6. customer can checkout with CARD

**Comment by KateChuen** [comment]
Tested on dev3, against dev16 backend.

- [x] first make sure the b2c checkout work well (same as before)
❌An issue when there's no address saved. Opened ticket #8012. Verified with Wei that it's not related to b2b checkout.
But works well when there's already an address saved. 
❌However, if we change the country for the shipping address, then we have another issue where it does not display the correct shipping methods. Opened ticket #8014. 

- [x] switch to b2b, make sure each step work well

- [x] change payment type, the checkout data will be cleared, and customer redirect to first step
❗️if we chose account payment at first, and added a PO number. then change the payment type to card payment, it goes back to first step but the PO number is not cleared. It stays until the order review page. Confirmed with Wei that it's the expected behavior. She says the backend tries to keep the PO number.

- [x] change language works well in each step

- [x] customer can checkout with ACCOUNT

- [x] customer can checkout with CARD
